### PR TITLE
Malicious tokens inside js file

### PR DESCRIPTION
**Pull request by hrajotia**

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

**Comment by dancgray** [comment]
+1

**Comment by taylortom** [comment]
Don't understand this PR. What's been done?

File: js/adapt-contrib-matching.js [hunk 1]
```diff
@@ -108,20 +108,19 @@ define(function(require) {
         storeUserAnswer: function() {
 
             var userAnswer = new Array(this.model.get('_items').length);
-+           var tempUserAnswer = new Array(this.model.get('_items').length);
```
Comment (taylortom) [inline js/adapt-contrib-matching.js]: Ah I see now

**Comment by taylortom** [comment]
+1

**Pull request merged**